db/migrate/20220425102039_remove_bill_addresses_with_null_phone.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveBillAddressesWithNullPhone < ActiveRecord::Migration[6.1]
+  def change
+    remove_foreign_key "spree_users", "spree_addresses", column: "bill_address_id", name: "spree_users_bill_address_id_fk"
+    add_foreign_key "spree_users", "spree_addresses", column: "bill_address_id", name: "spree_users_bill_address_id_fk", on_delete: :nullify
+    execute("DELETE from spree_addresses WHERE phone IS NULL AND id IN(SELECT bill_address_id FROM spree_users)")
```

Comment:
This is really cool! I need to get more creative next time.

Reply:
Except that it doesn't work. I forgot an `=` when assigning the table name. I thought that you would test the code instead of just copying it. Otherwise I could have done it myself. I'm just testing a few more things now and I'll commit a fixed version.